### ISSUE TITLE
Is there a way to just highlight the options in current view?

### ISSUE THREAD

**Issue by tristone13th** [AlexPl292/IdeaVim-EasyMotion#58]
When press the <leader><leader>e/w in any position, it will list all the options although some of them (or most) are out of the view (such as the last line), which results in a waste of characters. most time we must press two characters to jump to the target. Is there a way to just list all the options in the current view?

**Comment by SamShowalter** [comment]
Agreed, this is also something I find to be an issue. Is there any update? It would be nicer to restrict the selction and only pick single letter targets

**Comment by philfontaine** [comment]
Since this plugin is powered by AceJump, go to the AceJump settings and uncheck `Search whole file`.

![image](https://github.com/AlexPl292/IdeaVim-EasyMotion/assets/16233697/add43a2a-1583-4f11-b890-0f89e9278a34)
